### PR TITLE
Feature/41 images

### PR DESCRIPTION
**Pull request by jshor**
adds ability to upload/modify a user's image

### CODE REVIEW

**Comment by jackofseattle** [comment]
You have some merge conflicts here.

File: client/vivace/add-edit/controller.js [hunk 1]
```diff
@@ -4,38 +4,43 @@ module.exports = [
...
   '$routeParams',
   'CandidateModel',
-  function(regexProvider, $location, $routeParams, CandidateModel) {
+  'BlobConversion',
+  function(regexProvider, $location, $routeParams, CandidateModel, BlobConversion) {
 
     let promise;
 
-
+    this.defaultAvatar = 'http://i.imgur.com/AnSoiBG.png';
```
Comment (jackofseattle) [inline client/vivace/add-edit/controller.js]: Save it as empty, lets not store the default image in the server.

File: client/vivace/lib/blobConversion.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+/**
+ * Converts a blob reference to base64 data
+ */
+module.exports = [
+  '$http',
+  ($http) => {
+    let convert;
```
Comment (jackofseattle) [inline client/vivace/lib/blobConversion.js]: We don't usually define everything at the top of the scope. makes for weird reading.

**Comment by jshor** [comment]
@jackofseattle all merge conflicts fixed and tests added

**Comment by jackofseattle** [comment]
LGTM

**Pull request merged**